python/lsst/dc2gen3/patch_dc2.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+"""Modify a DESC DC2 to prepare it for conversion to gen3
+"""
+
+# imports
+
+import argparse
+import sys
+import os
+import shutil
+
+from astropy.io import fits
+import lsst.log
+import lsst.log.utils
+
+from lsst.daf.persistence import Policy
+from lsst.obs.base.gen2to3.repoWalker import PathElementParser
+
+# constants
+
+DATATYPES_TO_CONVERT = ("flat", "sky", "SKY")
+POLICY_FILE = Policy.defaultPolicyFile(
+    "obs_lsst", "lsstCamMapper.yaml", "policy"
+)
+FIX_HEADER = False
+
+# exception classes
+
+# interface functions
+
+
+def copy_files(origin_repo, new_repo, policy_file=POLICY_FILE):
+    """Copy files from a DESC DC2 gen2 origin repo to one with modified names
+    to change parsed filter names to match that expected by the gen3 butler.
+
+    Parameters
+    ==========
```

Comment:
For our numpydoc standard we would say

```
Parameters
-----------
origin_repo : `str`
    Full path...
```
etc (so backticks around types and `--` not `==`